packages/berlin/src/pages/Cycle.tsx

```diff
@@ -187,18 +187,34 @@ function Cycle() {
     }));
   };
 
+  let voteInfo = '';
+
+  switch (cycleState) {
+    case 'closed':
+      voteInfo = 'Vote has ended.';
+      break;
+    case 'upcoming':
+      voteInfo = `Vote opens in: ${formattedTime}`;
+      break;
+    case 'open':
+      if (time && time < 301) {
+        voteInfo = `Vote closes in: ${formattedTime}`;
+      } else if (time === 0) {
+        voteInfo = 'Vote has ended.';
+      }
+      break;
+    default:
+      voteInfo = '';
+  }
+
+  <Body>{voteInfo}</Body>;
+
```

Comment:
2 questions here:
1. how does the Body tag work here
2. why use a let and now a state variable? 
3. also save the 301 as a const with more description